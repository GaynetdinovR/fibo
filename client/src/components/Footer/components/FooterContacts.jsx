import styles from "../../../styles/components/Footer.module.sass";
import H6 from "../../../ui/H6.jsx";
import LinkInList from "../../../ui/LinkInList.jsx";

const FooterContacts = () => {
	const contacts = [
		{ src: "./icons/viber.png", alt: "viber" },
		{ src: "./icons/skype.png", alt: "skype" },
		{ src: "./icons/messenger.png", alt: "messenger" },
		{ src: "./icons/telegram.png", alt: "telegram" },
		{ src: "./icons/facebook.png", alt: "facebook" },
		{ src: "./icons/vk.png", alt: "vk" }
	];

	return (
		<nav className={styles.footer__contacts}>
			<H6 className={styles.footer__contacts_title}>
				Остались вопросы? А мы всегда на связи:
			</H6>
			<ul className={styles.footer__contacts_list}>
				{contacts.map((item, i) => (
					<LinkInList
						key={i}
						liClass={styles.footer__contacts_item}
						aClass={styles.footer__contacts_link_img}
					>
						<img src={item.src} alt={item.alt} />
					</LinkInList>
				))}
				<LinkInList
					liClass={styles.footer__contacts_item}
					aClass={styles.footer__contacts_link}
				>
					Написать нам
				</LinkInList>
			</ul>
		</nav>
	);
};

export default FooterContacts;