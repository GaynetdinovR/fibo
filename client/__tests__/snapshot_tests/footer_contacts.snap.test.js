import renderer from "react-test-renderer";
import { expect, it } from "@jest/globals";
import FooterContacts from "../../src/components/Footer/components/FooterContacts.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<FooterContacts />).toJSON();
	expect(tree).toMatchSnapshot();
});
