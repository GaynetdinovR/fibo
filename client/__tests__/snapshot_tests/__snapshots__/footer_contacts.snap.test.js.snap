// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<nav
  className="footer__contacts"
>
  <h6
    className="footer__contacts_title h6"
  >
    Остались вопросы? А мы всегда на связи:
  </h6>
  <ul
    className="footer__contacts_list"
  >
    <li
      className="footer__contacts_item link_in_list__item"
    >
      <a
        className="footer__contacts_link_img link_in_list__link"
      >
        <img
          alt="viber"
          src="./icons/viber.png"
        />
      </a>
    </li>
    <li
      className="footer__contacts_item link_in_list__item"
    >
      <a
        className="footer__contacts_link_img link_in_list__link"
      >
        <img
          alt="skype"
          src="./icons/skype.png"
        />
      </a>
    </li>
    <li
      className="footer__contacts_item link_in_list__item"
    >
      <a
        className="footer__contacts_link_img link_in_list__link"
      >
        <img
          alt="messenger"
          src="./icons/messenger.png"
        />
      </a>
    </li>
    <li
      className="footer__contacts_item link_in_list__item"
    >
      <a
        className="footer__contacts_link_img link_in_list__link"
      >
        <img
          alt="telegram"
          src="./icons/telegram.png"
        />
      </a>
    </li>
    <li
      className="footer__contacts_item link_in_list__item"
    >
      <a
        className="footer__contacts_link_img link_in_list__link"
      >
        <img
          alt="facebook"
          src="./icons/facebook.png"
        />
      </a>
    </li>
    <li
      className="footer__contacts_item link_in_list__item"
    >
      <a
        className="footer__contacts_link_img link_in_list__link"
      >
        <img
          alt="vk"
          src="./icons/vk.png"
        />
      </a>
    </li>
    <li
      className="footer__contacts_item link_in_list__item"
    >
      <a
        className="footer__contacts_link link_in_list__link"
      >
        Написать нам
      </a>
    </li>
  </ul>
</nav>
`;
